Packs/CheckPhish/Integrations/CheckPhish/CheckPhish_test.py

```diff
@@ -12,31 +11,19 @@
     'error': False
 }
 
-cases = [
-    (
-        'phish',
-        3
-    ),
-    (
-        'bad',
-        0
-    )
-]
 
-
-@pytest.mark.parametrize('bad_disp, expected_score', cases)
-def test_reliability_in_get_result_checkphish(requests_mock, mocker, bad_disp, expected_score):
+def test_reliability_in_get_result_checkphish(requests_mock, mocker):
     """
         Given:
-            - The user reliability param and the bad_disp param
+            - The user reliability param and the bad_disp param that exist in the default
         When:
             - Running get_result_checkphish
         Then:
             - Verify reliability as excepted
             - Verify dbot score is expected according the bad disposition
     """
     mocked_results = RESULTS
-    mocked_results['disposition'] = bad_disp
+    mocked_results['disposition'] = 'phish'
```

Comment:
why a `phish` disposition is a dbotsscore 3?, you would have to explian in the docstring

Reply:
I have written that the given bad_disp is in the default bad_disp dict, and the second test got a bad_disp which is not in the default